lib/venue.rb

```diff
@@ -1,2 +1,45 @@
 class Venue
+  # initialize read abilities for three major attributes
+  attr_reader :name, :capacity, :patrons
+
+  # initialize venue class
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  # add a patron to the patrons array
+  def add_patron(patron)
+    @patrons << patron
+  end
+
+  # return a list of patrons that is all uppercase
+  def yell_at_patrons
+    yelled_at_patrons = []
+    patrons.each do |patron|
+      yelled_at_patrons << patron.upcase
+    end
+    return yelled_at_patrons
```

Comment:
Nice job with the `.each`. You do not need the keyword `return` here. You can just put `yelled_at_patrons` as the last line